### PR TITLE
[WIP] Rename ultralight to lightest sync mode

### PR DESCRIPTION
**Pull request by timmoreton**
### Description

Geth 1.9 includes a new `ultralight` sync mode, so to avoid confusion and code clashes we have renamed ours to `lightest`. This is the companion PR for those changes.

### Backwards compatibility

No

### CODE REVIEW

File: packages/walletkit/local.tar.gz [hunk 1]
```diff
@@ -0,0 +1 @@
+No such object: contract_artifacts/local
```
Comment (jmrossy) [inline packages/walletkit/local.tar.gz:1]: Did you mean to check this in? Looks like an artifact of a failed build or something?

File: packages/walletkit/build/local/contracts [hunk 2]
```diff
@@ -0,0 +1 @@
+../protocol/build/contracts/
```
Comment (jmrossy) [inline packages/walletkit/build/local/contracts:1]: what's this change?

File: packages/web/static/locales/en/dev.json [hunk 3]
```diff
@@ -68,7 +68,7 @@
     "text":
       "Send or Request Celo Dollar easily with anyone in your address book, powered by decentralized phone number verification",
     "nonCustodial": "Non-custodial wallet",
-    "mobileUltra": "Mobile ultralight client",
+    "mobileUltra": "Mobile lightest client",
```
Comment (jmrossy) [inline packages/web/static/locales/en/dev.json:71]: hope we're certain we won't go lighter in the future, not leaving a lot of room with this new name haha ;)

Comment (aaronmgdr) [reply packages/web/static/locales/en/dev.json:71]: superduperlight?

File: packages/mobile/src/geth/geth.ts [hunk 4]
```diff
@@ -22,7 +22,7 @@ export const FailedToFetchGenesisBlockError = new Error(
 // We are never going to run mobile node in full or fast mode.
 enum SyncMode {
   LIGHT = 'light',
-  ULTRA_LIGHT = 'ultralight',
+  ULTRA_LIGHT = 'lightest',
```
Comment (tkporter) [inline packages/mobile/src/geth/geth.ts:25]: Intentional to keep this as `ULTRA_LIGHT`?

**Review by tkporter — APPROVED** [review]

**Comment by aaronmgdr** [comment]
close out?

**Pull request closed without merging**